ui/layout-pages.md

```diff
@@ -0,0 +1,133 @@
+
+These are controlled using the parameters on the column (`oh-grid-col`) components - you can configure them in the Design tab with the *Column Options* menu entries, or in YAML with the Code tab or a "Edit YAML" options on a parent component.
+
+![Column Options](./images/responsive1.png)
+
+Using code, you can quickly duplicate the breakpoints for similar columns.
+
+The breakpoints work like this: the `width` property of the column is the default width that will be applied for the smallest screen, then `xsmall`, `small`, `medium`, `large`, `xlarge` width will apply on wider screens appropriately:
+
+| Property | Minimum screen width | Examples of devices |
+|----------|----------------------|---------------------|
+| `width` | No minimum, default | All devices including phones |
+| `xsmall` | >= 480px | Smartphones in landscape mode |
+| `small` | >= 568px | Smaller tablets (Nexus 7...) in portrait mode |
+| `medium` | >= 768px | Most tablets in portrait mode |
+| `large` | >= 1024px | Smaller/regular tablets (iPad, iPad Mini...) in landscape mode* |
+| `xlarge` | >= 1200px | High-end tablets (iPad Pro...), desktops |
+
+<footnote><small>* Note that the sidebar will potentially be displayed and has a width of 260px</small></footnote>
+
+If you don't specify any width or breakpoints, the column will spread evenly on the row, without wrapping. This is fine for a couple of columns and simple widgets only, since on a small smartphone screen, you will rapidly get out of room.
+That's why you may find yourself wanting to let a column be 100% by default, so that it occupies the entire screen, and then reduce the width for `medium` or `large` breakpoints.
+
+For example, given this set of rows & cols:
+
+```yaml
+config:
+  label: Overview
+blocks:
+  - component: oh-block
+    config: {}
+    slots:
+      default:
+        - component: oh-grid-row
+          config: {}
+          slots:
+            default:
+              - component: oh-grid-col
+                config:
+                  width: "100"
+                  small: "50"
+                  medium: "33"
+                slots:
+                  default: []
+              - component: oh-grid-col
+                config:
+                  width: "100"
+                  small: "50"
+                  medium: "33"
+                slots:
+                  default: []
+              - component: oh-grid-col
+                config:
+                  width: "100"
+                  medium: "33"
+                slots:
+                  default: []
+```
+
+This is how the layout will adapt depending on the width of the screen:
+
+![Responsive Layout](./images/responsive2.gif)
+
+## How to Build a Layout Page
+
+### Defining the Layout
+
+When you create a new Layout Page, you'll notice the 2 buttons *"Add Block"* and *"Add Masonry"*.
```

Comment:
I would go for `Add Block` and `Add Masonry` (with single backticks) here.

That's how it is displayed in the preview:
![image](https://user-images.githubusercontent.com/7161177/106392155-e4fcb680-63f0-11eb-8f63-ea545b815e9e.png)

Reply:
Same remark as above. I think we should distinguish technical text and labels.